skimage/transform/radon_transform.py

```diff
@@ -262,6 +267,12 @@ def iradon(radon_image, theta=None, output_size=None,
     if filter not in filter_types:
         raise ValueError("Unknown filter: %s" % filter)
 
+    if preserve_range is None:
+        preserve_range = True
```

Comment:
Did the old behavior act as True? If so, I think we are trying to deprecate that

Reply:
In fact, previously `iradon` was accepting any input array data type and was casting it implicitly to `double`.
I remember now that I already saw in some discussion that `preserve_range` should be `False` by default.